pkg/applicationprofilemanager/v1/applicationprofile_manager.go

```diff
@@ -37,6 +38,8 @@ type ApplicationProfileManager struct {
 	execMaps                 maps.SafeMap[string, *maps.SafeMap[string, mapset.Set[string]]] // key is k8sContainerID
 	openMaps                 maps.SafeMap[string, *maps.SafeMap[string, mapset.Set[string]]] // key is k8sContainerID
 	watchedContainerChannels maps.SafeMap[string, chan error]                                // key is ContainerID
+	savedCapabilities        maps.SafeMap[string, int]
```

Comment:
Caveat: if we have several replica saving on the same profile, they will override syscalls